packages/ui/src/components/TokenValue.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import BN from 'bn.js'
+import styled from 'styled-components'
+import { Colors } from '../constants'
+import { formatTokenValue } from '../utils/formatters'
+
+interface props {
+  value: BN | number | undefined
+}
+
+export const TokenValue = React.memo(({ value }: props) => (
+  <TokenValueWrap>
+    {formatTokenValue(value)}&nbsp;<TokenSymbol>JOY</TokenSymbol>
+  </TokenValueWrap>
+))
+
+const TokenValueWrap = styled.span`
+  font-weight: 700;
+`
+
+const TokenSymbol = styled.span`
+  color: ${Colors.Black['400']};
```

Comment:
Not sure if we plan on having dark/light themes, but I'm used to having colors in the style components theme definition.

Reply:
That's sound reasonable. I'll track it here: https://github.com/Joystream/pioneer/issues/59.